drivers/include/shtc1.h

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2017, RWTH Aachen. All rights reserved.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_shtc1 SHTC1 Temperature and humidity sensor
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Driver for the Sensirion SHTC1 sensor.
+ *
+ * @file
+ * @brief       Device driver interface for the SHTC1 Temperature and humidity sensor
+ *
+ * @author      Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ */
+
+#ifndef SHTC1_H
+#define SHTC1_H
+
+#include <stdint.h>
+#include "saul.h"
+
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   SHTC1 Default Address
+ *
+ */
+#ifndef SHTC1_I2C_ADDRESS
+#define SHTC1_I2C_ADDRESS         (0x70)
+#endif
+
+
+/**
+ * @brief   enumeration to enable or disable the crc-check
+ * @{
+ */
+typedef enum {
+    CRC_DISABLED = 0,   /**< Disable CRC Check */
+    CRC_ENABLED         /**< Enable CRC Check */
+} shtc1_crc_type_t;
+
+/**
+ * @brief   value struct, where all results obtained from the sensor will be saved.
+ * @{
+ */
+typedef struct {
+    int16_t temp;          /**< Temperature in centi °C (2372 = 23.72) after an call to the measurement function. */
+    uint16_t rel_humidity; /**< Relative humidity in centi percent (9922 = 99.22) after an call to the measuerment function. */
+    unsigned int id;       /**< ID read from the sensor, only available after shtc1_id() was called. */
+} shtc1_values_t;
```

Comment:
Those are never used - and it doesn't really make sense to cache those values.
If you want to return both values without doing two reads over i2c, just add a 
```C
int8_t shtc1_read(const shtc1_t *dev, uint16_t *rel_humidity, uint16_t* temperature);
```
function.

Or maybe that's the only function and if you don't want either value you just pass `NULL`.